CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# v0.42.2 (_2019-10-21_)
+
+[Full Changelog](https://github.com/mozilla/application-services/compare/v0.42.1...v0.42.2)
+
+## Android
+
+### What's new
+
+- Exposed `stroage::bookmarks::erase_everything`, which deletes all bookmarks without affecting      history, through FFI.
```

Comment:
nit: stroage => storage, and affecting history has a lot of whitespace between it. (Also, a link to the PR would be nice...)